internal/workload/logging/model.go

```diff
@@ -65,7 +65,12 @@ func (l LogDestinationLoki) GrafanaURL(ctx context.Context) string {
 	const tpl = `var-ds=%s-loki&var-filters=service_name|%%3D|%s&var-filters=service_namespace|%%3D|%s`
 
 	tenantName := fromContext(ctx).tenantName
+	envName := l.EnvironmentName
+	// All loki logs are stored in gcp, update the envName to match the loki datasource
+	if len(envName) >= 4 && envName[len(envName)-4:] == "-fss" {
+		envName = envName[:len(envName)-4] + "-gcp"
```

Comment:
```suggestion
	if strings.HasSuffix(envName, "-fss") {
		envName = strings.TrimSuffix(envName, "-fss") + "-gcp"
```

Ikke en showstopper, Men synes personlig det leser litt lettere :)